web/libs/ui/src/lib/checkbox/checkbox.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Checkbox from "./checkbox";
+
+describe("Checkbox", () => {
+  it("should render successfully", () => {
+    const { baseElement } = render(<Checkbox />);
+    expect(baseElement).toBeTruthy();
+  });
+
+  it("should render with a label when children are provided", () => {
+    const { getByText } = render(<Checkbox>Test Label</Checkbox>);
+    expect(getByText("Test Label")).toBeTruthy();
+  });
+
+  it("should be checked when checked prop is true", () => {
+    const { getByRole } = render(<Checkbox checked={true} />);
+    expect((getByRole("checkbox") as HTMLInputElement).checked).toBe(true);
```

Comment:
I would add that in this case it's controlled and checked state should not be changed when we click. and for the next check as well.